relay-server/src/actors/store.rs

```diff
@@ -597,6 +658,8 @@ impl KafkaMessage {
             Self::UserReport(message) => message.event_id.0,
             Self::Session(message) => message.session_id,
             Self::Metric(_message) => Uuid::nil(), // TODO(ja): Determine a partitioning key
+            Self::ProfilingTrace(_message) => Uuid::nil(), // TODO(pierre): parse session_id as uuid
```

Comment:
Could you assign a random UUID here? Before putting load on this, I'll fix this for profiling and metrics.

Reply:
If I'm correct, a `uuidv4` will be assigned (https://github.com/getsentry/relay/pull/1170/files#diff-b043940a406beb64d66fa208a834b04f23942ae398a6cbe8633514e9ddff0eabR665-R666) if I return `nil` here. Do you want me to just be explicit about it?

Reply:
Yes, you're right. Thanks. I was looking for that and didn't see it right beneath :)